src/cosmic/evolve.py

```diff
@@ -349,7 +349,7 @@ def evolve(self, initialbinarytable, pool=None, bpp_columns=None, bcm_columns=No
                     path_to_tracks=initialbinarytable['path_to_tracks'].iloc[0], 
                     path_to_he_tracks=initialbinarytable['path_to_he_tracks'].iloc[0],
                     IBT_Z=initialbinarytable['metallicity'].iloc[0],
-                    z_accuracy_limit=z_accuracy_limit
+                    z_accuracy_limit=1e-2
```

Comment:
Should this be `z_accuracy_limit` instead?

Reply:
Realized after chatting with @d-maclean that this is only specified if SSEDict is not passed, so it is ok!